server/map_rpc_server.go

```diff
@@ -393,3 +359,14 @@ func (t *TrillianMapServer) getTreeAndHasher(ctx context.Context, treeID int64,
 	}
 	return tree, th, nil
 }
+
+// InitMap implements the RPC Method of the same name.
+func (t *TrillianMapServer) InitMap(ctx context.Context, req *trillian.InitMapRequest) (*trillian.InitMapResponse, error) {
```

Comment:
It's not clear why there are two `Init` functions in the map server...

Reply:
For minimal diff, really - `Init()` was pre-existing, but I've collapsed it into the RPC method now.

Reply:
Consider either collapsing the two functions -- I don't think the diff size is going to be that bad, or maybe make the pre-existing `Init` function private? It would be nice to have a single point of entry for this function.

Reply:
Reread the previous comment :)